src/ViewModel/HeroBanner.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace eLife\Patterns\ViewModel;
+
+use Assert\Assertion;
+use eLife\Patterns\ArrayAccessFromProperties;
+use eLife\Patterns\ArrayFromProperties;
+use eLife\Patterns\ViewModel;
+
+final class HeroBanner implements ViewModel
```

Comment:
We need a test for this. I would do a more thorough review but adding a test might catch some issues and simplify review. Look to the `CarouselItemTest` as an example of required coverage.

Reply:
@nlisgo I just committed the test.